src/deployment/azuredeploy.bicep

```diff
@@ -80,29 +75,16 @@ module operationalInsights 'bicep-templates/operational-insights.bicep' = {
   }
 }
 
-module linuxServerFarm 'bicep-templates/server-farms.bicep' = {
-  name: 'linux-server-farm'
+module serverFarm 'bicep-templates/server-farms.bicep' = {
+  name: 'server-farm'
   params: {
     server_farm_name: name
     owner: owner
     location: location
-    use_windows: false
-    create: true
-  }
-}
-
-module dotNetServerFarm 'bicep-templates/server-farms.bicep' = {
-  name: (enable_remote_debugging) ? 'windows-server-farm' : 'same-linux-server-farm'
-  params: {
-    server_farm_name: (enable_remote_debugging) ? '${name}-net' : name
-    owner: owner
-    location: location
     use_windows: enable_remote_debugging
```

Comment:
I think we decided that we should switch to Windows by default? We will need to test how the upgrade works though, it might require deleting/recreating the App Service Plan (serverFarm)

Reply:
Can we leave this change be made at a later time?

Reply:
It might be easier from a deployment perspective to keep the changes relegated to what's completely necessary.